test/modules/packages/Toml/test/fileTest.chpl

```diff
@@ -1,13 +1,14 @@
-
 use TOML;
 
 config const f: string;
 
 proc main() {
-  var error: syserr;
-  var TomlFile = open(error, f, iomode.r);
```

Comment:
huh, I'm not sure why this code used the error argument at all, since it didn't check it.
It seems to me simply removing the error argument is more reasonable in this case, at least based on what I see here.